aiven/client/client.py

```diff
@@ -101,27 +103,35 @@ def _execute(self, func, method, path, body, params=None):
 
         return response
 
-    def get(self, path="", params=None):
+    def get(self, path: str = "", params: Any = None) -> Response:
         """HTTP GET"""
         return self._execute(self.session.get, "GET", path, body=None, params=params)
 
-    def patch(self, path="", body=None, params=None):
+    def patch(self, path: str = "", body: Any = None, params: Any = None) -> Response:
         """HTTP PATCH"""
         return self._execute(self.session.patch, "PATCH", path, body, params)
 
-    def post(self, path="", body=None, params=None):
+    def post(self, path: str = "", body: Any = None, params: Any = None) -> Response:
         """HTTP POST"""
         return self._execute(self.session.post, "POST", path, body, params)
 
-    def put(self, path="", body=None, params=None):
+    def put(self, path: str = "", body: Any = None, params: Any = None) -> Response:
         """HTTP PUT"""
         return self._execute(self.session.put, "PUT", path, body, params)
 
-    def delete(self, path="", body=None, params=None):
+    def delete(self, path: str = "", body: Any = None, params: Any = None) -> Response:
         """HTTP DELETE"""
         return self._execute(self.session.delete, "DELETE", path, body, params)
 
-    def verify(self, op, path, body=None, params=None, result_key=None, retry=None):
+    def verify(
+        self,
+        op: Callable[..., Response],
+        path: str,
+        body: Any = None,
+        params: Any = None,
+        result_key: Optional[str] = None,
+        retry: Optional[int] = None
+    ) -> Dict[Any, Any]:
```

Comment:
I think it's better to return `Mapping[Any, Any]` to avoid surprises causes by mutations of the result. So, all other places which call `verify` need to be updated too.